ginkgo/outline/_testdata/position_test.go.csv

```diff
@@ -1,3 +1,3 @@
 Name,Text,Start,End,Spec,Focused,Pending,Labels
-Describe,104,240,104,240,false,false,false,""
-It,209,236,209,236,true,false,false,""
+Describe,"104,240",104,240,false,false,false,
+It,"209,236",209,236,true,false,false,
```

Comment:
I laughed to discover that `ginkgo/outline/_testdata/position_test.go` already has text in place which should have forced using a CSV writer. The test in question:

https://github.com/onsi/ginkgo/blob/9e2f337b10cf4973bbc9ae4aeac5a3a334c3571a/ginkgo/outline/_testdata/position_test.go#L1-L22

A test which uses a csv Reader to load each `_testdata/*_test.go.csv` would have caught this, but it's easy to miss out on writing meta-tests.

In order not to clutter this PR, I have not added such a test to the suite. If the reviewers like, I can add such a commit onto this branch; or provide a separate PR.

Reply:
Here's a picture to hopefully illustrate the previous error in this file:

![image](https://github.com/user-attachments/assets/8fd70bfb-0279-48ed-9c95-93fb02f93e7c)

Counting the comma-separated fields in each line, you'll see:

|Comment|Row|Fields|
|-|-|-|
|Header|`Name,Text,Start,End,Spec,Focused,Pending,Labels`|8|
|Removed line|`Describe,104,240,104,240,false,false,false,""`|9|
|Removed line|`It,209,236,209,236,true,false,false,""`|9|
|Added line|`Describe,"104,240",104,240,false,false,false,`|8|
|Added line|`It,"209,236",209,236,true,false,false,`|8|

Reply:
ha! this is great, thanks. yes please feel free to add that test to this PR.

Reply:
Done!

I wrote the test and saw it pass on the HEAD of my branch. I rebased it to pretend I'd written this meta-test as the first, rather than last, commit on this branch.

Then I just checked out each commit and ran

     belden@laptop github/onsi/ginkgo/ginkgo/outline $ go test .

to see how the test failed. With the meta-test as the first commit, it properly fails due to `outline/_testdata/position_test.go.csv` having the extra fields, as shewn in the screenshot above.

The test continues to fail in various ways for the subsequent commits; and when we finally get to the HEAD of this branch, the test passes as expected.

----

I broke the test up into two functions so as not to make the `DescribeTable` test body be too long. When I initially looked at that file, I immediately saw the super helpful comment about how to extend the test. I wanted to let the next person immediately see that same helpful comment.